splice_spark/src/main/spark2.1/com/splicemachine/spark/splicemachine/SplicemachineContext.scala

```diff
@@ -325,6 +325,28 @@ class SplicemachineContext(options: Map[String, String]) extends Serializable {
     internalConnection.createStatement().executeUpdate(sqlText)
   }
 
+  /**
+    *
+    * Sample the dataframe, split the table, and insert a dataFrame into a table (schema.table).  This corresponds to an
+    *
+    * insert into from select statement
+    *
+    * @param dataFrame
+    * @param schemaTableName
+    * @param sampleFraction
+    */
+  def insert(dataFrame: DataFrame, schemaTableName: String, sampleFraction: Double): Unit = {
```

Comment:
I would name this method something more explicit: splitAndInsert, insertAfterSplitting, insertWithSplits ...

Reply:
API was changed to splitAndInsert